zstacklib/zstacklib/utils/iscsi_nl.py

```
@@ -0,0 +1,484 @@
+#!/usr/bin/env python2
+# generated by o1-mini
+
+import struct
+import logging
+import os
+import sys
+import re
+import socket
+import errno
```

Ruff (0.11.9)

`errno` imported but unused

Remove unused import: `errno`

(F401)